src/engine-actions/translate-swap.js

```diff
@@ -44,6 +44,84 @@ function calculateTimeLock (extendedTimeLockDelta, secondsPerBlock, blockHeight)
   return maxTimeLockDeltaInBlocks.plus(blockHeight).toString()
 }
 
+/**
+ * gRPC error code for a CANCELLED request, either by a client or a server
+ * @see {@link https://github.com/grpc/grpc/blob/master/doc/statuscodes.md}
+ * @constant
+ * @type {Number}
+ */
+const CANCELLED_CODE = 1
+
+/**
+ * Use the sendToRoute function as though it returned a promise
+ * @param  {String}      paymentHash     Base64 string of the payment hash to pay for
```

Comment:
Shouldn't this be `bytes` instead? looks like the call to sendToRoute either accepts paymentHash or paymentHashString

Reply:
our settings on grpc convert Base64 strings to `bytes`, so the description is accurate.